tests/test_lib_smsprovider.py

```diff
@@ -256,6 +267,35 @@ def test_08_smsgateway_success(self):
         r = sms.submit_message("123456", "Halo")
         self.assertTrue(r)
 
+    @smtpmock.activate
+    def test_09_send_sms_regexp_success(self):
+        smtpmock.setdata(response={"recp@example.com": (200, "OK")})
+        # Here we need to send the SMS
+        r = self.regexp_provider.submit_message("+49 123/456-78", "Hello")
+        self.assertTrue(r)
```

Comment:
Unfortunately this does not really check if the regexp was (successfully) applied. I think we need some consistent logging when submitting an SMS.
We can keep it like that for now.